src/Test/TestAIMuriel.java

```diff
@@ -12,29 +17,30 @@
 import Terrain.MapLibrary;
 import Terrain.MapMaster;
 import Terrain.TerrainType;
+import Terrain.Maps.MapReader;
 import Units.Unit;
 
 public class TestAIMuriel extends TestCase
 {
   private static Commander testCo1;
   private static Commander testCo2;
-  private static Muriel testAI;
+  private static AIMaker[] ais = { Muriel.info, WallyAI.info };
```

Comment:
No love for ISAI/Spender?

Reply:
They'd just fail all the tests anyway, so...

Reply:
They would. But it would be funny.

Reply:
It might be funny, if it didn't cause a test failure and prevent starting the game, lol